nixos/modules/services/monitoring/collectd.nix

```diff
@@ -98,7 +98,7 @@ in {
       description = ''
         Attribute set of plugin names to plugin config segments
       '';
-      type = types.attrsOf types.str;
+      type = attrsOf lines;
```

Comment:
Does it make sense to merge cpu, memory and network?

```
Server 192.168.1.1 25826
Server 192.168.1.2 25826
Server 192.168.1.3 25826
```

Reply:
It's still `attrsOf lines` so the lines get merged per plugin not all.

Reply:
Any plugin's configuration must be enclosed by `<Plugin ...>`[...]`</Plugin>` in the final collectd.conf so we have to keep the plugin settings per-plugin.